File: backend/src/models/DBANGGIATHEOKHUVUC.ts
import { Model, DataTypes } from 'sequelize';
import db from '../config/database';

class DBANGGIATHEOKHUVUC extends Model {
    public id!: any;
    public note!: any;
    public status!: any;
    public usermodifiedid!: any;
    public timemodified!: any;
    public timecreated!: any;
    public dkhuvucid!: any;
    public usercreatedid!: any;
    public dbanggiaid!: any;
}

DBANGGIATHEOKHUVUC.init({
    ID: {
        type: DataTypes.INTEGER,
        primaryKey: true, autoIncrement: true,
    },
    NOTE: {
        type: DataTypes.STRING,
        
    },
    STATUS: {
        type: DataTypes.BOOLEAN,
        
    },
    USERMODIFIEDID: {
        type: DataTypes.INTEGER,
        
    },
    TIMEMODIFIED: {
        type: DataTypes.DATE,
        
    },
    TIMECREATED: {
        type: DataTypes.DATE,
        
    },
    DKHUVUCID: {
        type: DataTypes.INTEGER,
        
    },
    USERCREATEDID: {
        type: DataTypes.INTEGER,
        
    },
    DBANGGIAID: {
        type: DataTypes.INTEGER,
        
    },
}, {
    sequelize: db.sequelize,
    modelName: 'DBANGGIATHEOKHUVUC',
    tableName: 'DBANGGIATHEOKHUVUC',
    timestamps: false
});

export default DBANGGIATHEOKHUVUC;

File: frontend/Models/DBANGGIATHEOKHUVUC.cs
using System;

namespace QuanLyBar.Client.Models
{
    public class DBANGGIATHEOKHUVUC
    {
        public int? Id { get; set; }
        public string Note { get; set; }
        public bool? Status { get; set; }
        public int? UsermodifiedId { get; set; }
        public DateTime? Timemodified { get; set; }
        public DateTime? Timecreated { get; set; }
        public int? DkhuvucId { get; set; }
        public int? UsercreatedId { get; set; }
        public int? DbanggiaId { get; set; }
    }
}
